pyracf/common/security_result.py

```diff
@@ -130,6 +143,12 @@ def __to_pascal_case(self, key: str) -> str:
             case _:
                 return key
 
+    def __get_current_userid(self) -> str:
+        if platform.system() == "OS/390":
+            return getpass.getuser().lower()
+        else:
+            return "testuser"
+
```

Comment:
I think it is better to create a mock in the unit tests to do this instead of adding this to the pyracf source code.

I think you should be able to do something like this in `test_running.py`.

```python
@patch('getpass.getuser')
def __test_suite(get_user_mock: Mock) -> unittest.TestSuite:
    """Load and run each unit test class."""
    get_user_mock.return_value = "patrick"
    test_suite = unittest.TestSuite()
    test_loader = unittest.TestLoader()
```

I think that mock should then be inherited by all of the test suites.

Reply:
That's fair, I avoided this because I thought it would be a lot to handle for a feature that could never be used outside of z/OS anyway (it's not in the request builder functionality), but I can try to make this adjustment.

Reply:
It is generally good to avoid having code that is only ever used for testing in the source code. The only exception in our case is the encoding stuff, which I think is okay.

Reply:
I am having trouble getting it to it to inherit this sort of mock case. We'll have to discuss next week.

Reply:
I could not get patch to inherit the way we wanted, but forcing     `getpass.getuser = Mock(return_value="testuser")` in the test_suite worked well.